docs/source/api/index.md

```diff
@@ -13,7 +13,7 @@
   model_config
   model_builder
   prior
-  product_incrementality
+  cc
```

Comment:
Don't think this should be abbreviated

Reply:
as in the module name should be `customer_choice`, changing `cc` to `customer_choice` globally?

Reply:
yes